index.js

```diff
@@ -87,15 +94,7 @@ function coffeeify(filename, options) {
         header: false
     };
 
-    for (var i = 0, keys = Object.keys(compileOptions); i < keys.length; i++) {
-        var key = keys[i], option = options[key];
-        if (typeof option !== 'undefined' && option !== null) {
-            if (option === 'false' || option === 'no' || option === '0') {
-                option = false;
-            }
-            compileOptions[key] = !!option;
-        }
-    }
+    assign(compileOptions, options);
```

Comment:
Essentially this is Object.assign and will not be needed once Node is fully ES6 .